client/core/notification.go

```diff
@@ -100,6 +100,8 @@ type Notification interface {
 // FeePaymentNote is a notification regarding registration fee payment.
 type FeePaymentNote struct {
 	db.Notification
+	ConfirmationsRequired uint32 `json:"confirmationsrequired,omitempty"`
+	ConfirmationsReceived uint32 `json:"confirmationsreceived,omitempty"`
```

Comment:
Can drop the `Received` from the field name and json name.

Reply:
Good catch, done.